modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartClientCodegen.java

```diff
@@ -178,7 +192,7 @@ public String getName() {
 
     @Override
     public String getHelp() {
-        return "Generates a Dart (1.x (deprecated) or 2.x) client library.";
+        return "Generates a Dart client library.";
```

Comment:
Minor suggestion to revise as follows to make it clear the output is for Dart 2.x:
```
Generates a Dart 2.x client library.
```

Reply:
Why make it dart2 specific while dart 1.x support is being removed?

Reply:
To avoid any surprise to new users who may expect the output to work with Dart 1.x.